alf/environments/carla_controller.py

```diff
@@ -63,7 +63,7 @@ def step(self, current, target):
         else:
             derivative = 0.
         self._prev_error = error
-        return self._K_P * error + self._K_I * self._integral + self._K_P * derivative
+        return self._K_P * error + self._K_I * self._integral + self._K_D * derivative
```

Comment:
Does this affect the performance of using PID controller in CARLA?

Reply:
I'm running experiments using the fix one.